frontend/src/constants/hackerpack-partners.js

```diff
@@ -0,0 +1,32 @@
+const Partners = [
+  {
+    name: "Microsoft Azure",
+    description: `Microsoft Azure is a flexible, scalable, and full-featured cloud platform. If you can imagine it, you can build it on Azure, using the tools, languages and frameworks you're already familiar with. Python, JavaScript, Java, .NET - you can run them all on Azure. 
+
+Using Azure you can run serverless code using Azure Functions, host web applications and databases, take advantage of artificial intelligence (AI) through Cognitive Services, and create your own machine learning models through Azure Notebooks or powerful Data Science Virtual Machines (DSVM)`,
+    icon: "",
+    link: ""
+  },
+  {
+    name: "SUSE",
+    description: `amksdklamskldmakskl`,
+    icon: "",
+    link: ""
+  },
+  {
+    name: "Aiven",
+    description: `Aiven enables you to accelerate application innovation through a modern, fully-managed data infrastructure. From Aiven platform you can deploy any and all of our 8 open-source services, Apache Kafka and Cassandra, Elasticsearch, PostgreSQL, MySQL, Redis, InfluxDB, and Grafana in the cloud and region of your choice.
+
+All of this can be accomplished with a few button clicks. Launch your first service and get back to focusing on your project, not managing its underlying data infrastructure.`,
+    icon: "https://aiven.io/assets/img/press/aiven-logo_RGB.png",
+    link: "https://aiven.io/landing/junction"
+  },
+  {
+    name: "46elks",
+    description: `Send and receive SMS, MMS and Phone calls with just a few lines of code. Create an account at 46elks.com/junction and you’ll get free credits during the hackathon. You ask us questions using help@46elks.com, or come by our table at Junction.`,
```

Comment:
Let's remove the link from the description here so people don't bypass the link tracking :D

Reply:
I talked about this with the client, he'll be in contact with you today :D